st2common/st2common/rbac/utils.py

```diff
@@ -63,158 +56,82 @@
 ]
 
 
-def request_user_is_admin(request):
```

Comment:
Actually, now that I have checked more code - those functions were intended and there as convenience wrappers, but If removing them and directly retrieving user inside API controller some how makes it easier for you to make your other changes I'm fine with it (but would still like to hear reasoning :)).